src/commands/deck.ts

```diff
@@ -285,6 +300,19 @@ export class DeckCommand extends Command {
 		this.logger.info(serializeCommand(interaction), error);
 	}
 
+	protected async upload(filename: string, deck: Buffer): Promise<void> {
+		// specifying the port is a bit tricky bc typescript assumes envvars are strings
+		// for now, the default should be correct
+		await ftp.access({
+			host: process.env.FTP_HOST,
+			user: process.env.FTP_USER,
+			password: process.env.FTP_PASS,
```

Comment:
Might be easier for secret management if you provide the secret as one URL `ftp://user:pass@host:port` or similar, and then parse that to obtain the components.